packages/custom-functions-metadata-plugin/src/loader.js

```diff
@@ -0,0 +1,9 @@
+const { getOptions } = require('loader-utils');
```

Comment:
It would be great for this file to be of .ts typescript to be consistent with the rest of the project.
Just need to change .js to .ts and add the type for the variables.